vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -416,6 +421,41 @@ private void unstashWorkspace(config, prefix) {
     }
 }
 
+private List getInitContainer(config){
+    def result = []
```

Comment:
May be could give proper name for this variable.

Reply:
variable changed to initContainerSpecList